### DISCUSSION TITLE
Awesome !!

### DISCUSSION THREAD

**Discussion by MrAdityaAlok** [community/community#2885] [Discussions] [answered]
Awesome feature of GitHub.💐

**Comment by evi-liu** [comment, marked as answer]
Glad you're loving it! What's your favorite part so far about Discussions?

**Reply by MrAdityaAlok** [reply]
Well ! I do not have to create an **issue** for asking a question . ☺️
It's an awesome improvement 👍

**Reply by nbrugger-tgm** [reply]
It encourages discussion between users and is not like answers towards the devs only
